### DISCUSSION TITLE
How does one convert mouse sensitivity too tablet area size

### DISCUSSION THREAD

**Discussion by Leawur** [ppy/osu#25950] [Q&A]
In osu stable i use 1.18x but on osu lazer using mouse precision doesn't allow my tablet too work i'm using opentabletdrivers and would prefer to keep using them. i understand that all i have to do is make my tablet area smaller but i have no idea how to work that out.
![image](https://github.com/ppy/osu/assets/154428707/56fa4c1d-36fe-4eef-8f6c-dafc5f91f85d) 

Any help would be appreciated.

**Comment by GopoBoi** [comment]
just dividing the dimensions by the sensitivity should work. in this case that would be:
66.22 / 1.18 = 56.12
44.7 / 1.18 = 37.88

**Reply by Leawur** [reply]
thank you very much
